helpers.php

```diff
@@ -81,7 +81,9 @@ public function pull($branch = false)
 
     public function kirbyChange($commitMessage)
     {
-        $this->getRepo()->checkout($this->branch);
+        if ($this->branch && $this->branch != '') {
```

Comment:
it’s possible to remove `&& $this->branch != ''` here because if the branch is empty `if($this->branch)` is already false since we don’t check strict equality here.